### PR TITLE
Add Bos_2018 dataset

### PR DESCRIPTION
**Pull request by terrymyc**

### CODE REVIEW

File: datasets/Bos et al. (2018)/process_Bos_dataset.ipynb [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# retain useful columns\n",
+    "use_col = [\"title\", \"abstract\", \"keywords\", \"authors\", \"year\", \"doi\"]\n",
```
Comment (GerbrichFerdinands) [inline datasets/Bos et al. (2018)/process_Bos_dataset.ipynb:87]: Is it possible to keep the ID column from the original data as well?

**Comment by terrymyc** [comment]
> Could you still add information on this dataset the README of the main directory?

README will be updated later, along with corrected information on existing datasets.

> Also, I see the number of inclusions in the ris file (10) don't add up with the number of inclusions mentioned in the paper (11). I will consult the researchers about this inconsistency.

Thanks. The `.ris` file of inclusions on OSF only has 10 records for now.

**Comment by GerbrichFerdinands** [comment]
> Thanks. The `.ris` file of inclusions on OSF only has 10 records for now.

I checked, it is correct that there are 10 records. 
The study by Kaffashian et al. (2016) is used for 2 endpoints, hence the inconsistency.

**Comment by J535D165** [comment]
Thanks!

**Pull request merged**